coalib/io/FileFactory.py

```diff
@@ -0,0 +1,111 @@
+import linecache
+import os
+
+from coala_utils.decorators import generate_eq
+
+
+@generate_eq('name', 'timestamp')
+class FileFactory:
+    """
+    The FileFactory is a wrapper class to replace the actual file contents
+    from the file-dict. The object contains the following information about
+    the file:
+
+        * The filename (absolute file path).
+        * The modification timestamp of the file.
+        * The file in the form of a string.
+        * The file in the form of a list.
```

Comment:
not tuple? ;)
Btw, I would also add that it's `UTF-8` decoded, that's the difference between strings/lists and raw format ;)